src/scss/themes/_dark.scss

```diff
@@ -12,8 +12,8 @@
 
   --deemphasized-text-color: #{$deemphasized-color};
 
-  --very-deemphasized-link-color: #{rgba($anchor-color, 0.8)};
-  --very-deemphasized-text-color: #{lighten($main-bg-color, 32%)};
+  --very-deemphasized-text-color: #{lighten($main-bg-color, 44%)};
+  --very-deemphasized-link-color: var(--very-deemphasized-text-color);
```

Comment:
`--very-deemphasized-link-color` is redundant now that it's the same as `--very-deemphasized-text-color`. And it's only used in one place:

https://github.com/nolanlawson/pinafore/blob/774aa7a21c706d62a8ac08510c933ebaf5376eca/src/routes/_components/status/StatusContent.html#L66-L68

The impact of this is that links only have a color that differentiates them from the surrounding text if they're in a mention notification rather than a fav/boost notification:

![Screenshot from 2022-12-17 09-36-50](https://user-images.githubusercontent.com/283842/208254375-b02e6e5f-4f23-426c-b1ee-3b391745b749.png)

Technically this does not meet [WCAG link requirements](https://webaim.org/blog/wcag-2-0-and-link-colors/) since the only thing distinguishing the link is that they're underlined when you hover over them.

To be fair, though, the existing colors have the exact same problem:

![Screenshot from 2022-12-17 09-42-45](https://user-images.githubusercontent.com/283842/208254538-cd4bf7fd-0fc0-4469-9c22-25527254d8f0.png)

Maybe it would be best to just use the same (blue) color as links in mention notifications? Or a slightly different shade of blue? We don't need to solve this in this PR, though, since it's a pre-existing issue.

Reply:
One approach might be to just turn links into plain text for these types of notifications and make it one big clickable area, but agreed existing issue that could be improved.